api/bus.go

```diff
@@ -565,3 +565,17 @@ func FormatETag(ETag string) string {
 func ObjectPathEscape(path string) string {
 	return url.PathEscape(strings.TrimPrefix(path, "/"))
 }
+
+type (
+	ContractSetMetric struct {
+		Contracts int       `json:"contracts"`
+		Name      string    `json:"name"`
+		Time      time.Time `json:"time"`
+	}
+
+	ContractSetMetricsQueryOpts struct {
+		Name   *string
```

Comment:
Are the pointers necessary? Defaults would've worked fine too? Only asking because using pointers here makes it weird we're not using pointers in some other opts types.